### ISSUE TITLE
[LCD] Mesh Count not reset after emergency stop

### ISSUE THREAD

**Issue by Beagon** [OpenNeptune3D/OpenNept4une#163]
Hi, I had something happen and did an emergency stop during the creation of a KAMP mesh and had the following happen the next time I tried to print:

![IMG_20240401_192242097_HDR](https://github.com/OpenNeptune3D/OpenNept4une/assets/2945583/90306ab5-724d-4f5a-96f1-b1e066ffd3a9)

**Printer: Neptune 4 Plus**

Steps to reproduce:

1. Start a print with KAMP.
2. Let KAMP do a couple probes.
3. Emergency Stop the printer.
4. FIRMWARE_RESTART
5. Start print with KAMP again. Should start counting from the last probe it left off.

This seems to only be graphical.

**Comment by leokeba** [comment]
I have noticed this issue as well, but it seems more general than this case. Any probe command that was not part of the bed mesh procedure will have this effect, I often use the BED_LEVEL_SCREWS_TUNE macro before starting a print and it triggers the same issue.

**Comment by phillipthelen** [comment]
Should be fixed in dev
